packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -555,7 +554,6 @@ export const PersonalDetailsForm = ({
                 dirty,
             }) => (
                 <React.Fragment>
-                    {Notifications && <Notifications />}
```

Comment:
Why this is removed from personal details component?
Could there be notifications which should be only displayed when the user is in personal details route?

Reply:
i removed it from this component and added in parent account/app.jsx .